cmd/util.go

```diff
@@ -107,7 +107,7 @@ func createStack(
 	// stacks and well as httpstate stacks that opted into this by passing --secrets-provider passphrase
 	// while initialing a stack).  The only other supported provider today (the provider that uses the pulumi service
 	// does not need to be initialized explicitly, as creating the stack inside the Pulumi service does this).
-	if _, ok := b.(filestate.Backend); ok || secretsProvider == "passphrase" {
+	if secretsProvider == "passphrase" {
```

Comment:
We need to also do this in the case that the filestate backend is being used *and the secrets provider is “” or “default”*.   That’s because the default when using the filestate backend is the passphrase provider.